app.js

```diff
@@ -23,18 +23,19 @@
       Stripe.setPublishableKey(ENV.STRIPE_PUBLISHABLE_KEY);
     })
 
-    .controller('Controller', ['$scope', '$kinvey', '$http', '$interval', '$q',
-      function($scope, $kinvey, $http, $interval, $q) {
+    .controller('Controller', ['$scope', '$kinvey', '$http', '$interval', '$q', '$sce',
+      function($scope, $kinvey, $http, $interval, $q, $sce) {
         $scope.init = function() {
           $scope.user = $kinvey.getActiveUser();
           $scope.applicationErrors = [];
           $scope.isProcessingPayment = false;
           $scope.card = {
-             //number: '4242 4242 4242 4242',
-             //expMonth: '12',
-             //expYear: 2015,
-             //cvc: '232'
+            //number: '4242 4242 4242 4242',
+            //expMonth: '12',
+            //expYear: 2015,
+            //cvc: '232'
           };
+          $scope.livereloadUrl = $sce.trustAsResourceUrl('//localhost:35729/livereload.js');
```

Comment:
Adds live reload url to scope.  We only add it to the script tag when not in production now.  This keeps livereload from running on production.